Clients/src/presentation/components/AddNewRiskForm/RisksSection/index.tsx

```
@@ -11,313 +18,354 @@ import React from "react";
+              }}
+            >
+              <Field
+                id="risk-name-input"
+                label="Project name"
+                placeholder="Write risk name"
+                value={values.riskName}
+                onChange={handleOnTextFieldChange("riskName")}
+                sx={{
+                  gridRow: "1 / 2",
+                  gridColumn: "1 / 2",
+                }}
```

Biome (1.9.4)

[error] This JSX property is assigned multiple times.

This attribute is assigned again here.


(lint/suspicious/noDuplicateJsxProps)